files/nginx.upstreams.conf.template

```diff
@@ -0,0 +1,18 @@
+upstream monitor {
+    server 127.0.0.1:8887;
+}
+
+{% for application in applications %}
+# {{ application.name }}
+
+upstream {{ application.name }}_node {
+    server 127.0.0.1:{{ application.node_server_port }};
+    keepalive 8;
```

Comment:
Est-ce qu'il y a une raison historique pour mettre `keepalive` à `8` ? Il n'y a pas de valeur par défaut mais je me demandais si c'était un choix arbitraire ou non

À noter que l'on pourrait aussi faire des tests avec les timeouts vu que les erreurs type https://sentry.io/organizations/betagouv-f7/issues/2331718499/?project=5709078&query=is%3Aunresolved ou https://sentry.io/organizations/betagouv-f7/issues/3167697558/?project=5709078&query=is%3Aunresolved peuvent être causé par la config par défaut de nginx

Reply:
C'est un choix historique, je n'ai pas le souvenir de raison. 8 me parait bien bas (en fait je sais pas cf. https://www.nginx.com/blog/avoiding-top-10-nginx-configuration-mistakes#no-keepalives).

https://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive